performance-testing/uid2-core/k6-test-core-static.js

```diff
@@ -0,0 +1,150 @@
+import http from 'k6/http';
+
+const CORE_API_TOKEN = "UID2-O-L-999-dp9Dt0.JVoGpynN4J8nMA7FxmzsavxJa8B9H74y9xdEE=";
+const CORE_BASE_URL = 'http://localhost:8088';
+
+const AUTH_HEADERS = { headers: { 'Authorization': `Bearer ${CORE_API_TOKEN}` }};
+const HEADERS = { headers: { ...AUTH_HEADERS.headers, 'X-UID2-AppVersion': 'uid2-operator=0.0.0' }};
```

Comment:
Perhaps worth including something like "load-test" in X-UID2-AppVersion to distinguish it from real traffic